mmdet/datasets/pipelines/transforms.py

```diff
@@ -938,3 +938,188 @@ def __call__(self, results):
     def __repr__(self):
         repr_str = self.__class__.__name__ + f'(transforms={self.transforms})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class RandomCenterCropPad(object):
+    """Random center crop and random around padding for CornerNet.
+
```

Comment:
Add a detail description in Notes, e.g., how it works, what is the difference between `RandomCenterCropPad` and `RandomCrop` or other modules. A figure in docstring like below should also help (if I correctly understand how it works).

```python
"""
:code-block:
                   +----------------------------+
                    |                  padded area        |
    +--------|----------------------------|----------+
     |              |         cropped area                |                 |
     |              |         +---------------+        |      image |
     |              |         |        center  .     |         |                 |
     |              |         |        range         |         |                 |
     |              |         +---------------+        |                 |
    +--------|----------------------------|----------+
                    |                  padded area        |
                   +---------------------------+
"""
    

```